### ISSUE TITLE
Import map: unable to load a local module

### ISSUE THREAD

**Issue by FernandoBasso** [fannheyward/coc-deno#181]
I have `import-map.json` at the root directory of my project with this content:

```json
{
  "imports": {
    "/": "./",
    "./": "./"
  }
}
```

This is the contents of `.vim/coc-settings.json`:

```json
{
  "deno.enable": true,
  "deno.importMap": "./import-map.json",
  "tsserver.enable": false
}
```

And in some given test file, I have this as the first line:

```ts
import { assertEquals } from "/deps.ts";
```

That line causes the following error:

```
Unable to load a local module: "file:///deps.ts".
  Please check the file path. (deno no-local)
```

I cannot do `gf` to that file, for instance, and get no types or documentation whatsoever for `assertEquals`.

Running `deno test --import-map ./import-map.json src/foo.test.ts` works.

Is this an issue with my config/setup perhaps, or could it be some other problem?

**Comment by fannheyward** [comment]
Reproduced.

**Comment by FernandoBasso** [comment]
> It's `import_map.json`, not `import-map.json`.

This is what I have in `.vim/coc-settings.json`:

```
"deno.importMap": "./import-map.json"
```
Are you therefore implying that config is not honored?

Yet, renamed it to `import_map.json`, updated `coc-settings.json` accordingly, restarted vim and I still observe the same problems reported in the opening message.

I may have something else causing the problem, but I can't think of what it could be.

**Comment by fannheyward** [comment]
<img width="535" alt="截屏2022-10-10 20 07 34" src="https://user-images.githubusercontent.com/345274/194862912-4fc10dd8-1ca8-404a-8827-fd2e43a761df.png">

Have no idea about this, I just change to `import_map.json`, everything works as expected.

Test steps:

```
deps.ts
import_map.json
src/
src/foo.test.ts
```

`cat deps.ts`: `export * from "https://deno.land/std@0.156.0/testing/asserts.ts";`

**Comment by fannheyward** [comment]
Checked my configurations and found that I've global `"deno.importMap": "./import_map.json"`, that's why change file name work for me.

**Comment by FernandoBasso** [comment]
Oh yeah! If from vim I do `:CocConfig` (which edits the global config) an add  `"deno.importMap": "./import_map.json"` then it works.

![image](https://user-images.githubusercontent.com/6560620/194867605-79aab085-21c4-49a1-814b-5cc424596c3c.png)

Also, from the global config, `import-map.json` works too if that is how my import map is named. So, it looks like the local config is not being honored regarding the import map (it is working for enabling and disabling deno, for instance).

**Comment by fannheyward** [comment]
Still working on this, maybe there's something wrong on workspace configuration usage.

**Comment by fannheyward** [comment]
Fixed in v3.10.2

**Comment by FernandoBasso** [comment]
Confirm it is working! Great! Thanks!!
